src/main.tsx

```diff
@@ -349,7 +335,12 @@ Devvit.addSchedulerJob({
 
       if (!cachedForm) {
         console.error(`No cached form found for postId: ${postId}`);
-        await removePostAndFormFromRedis(redis, postId);
+        continue;
+      }
+
+      const status = cachedForm.getStatus();
+      if (status === FundraiserStatus.Completed || status === FundraiserStatus.Expired) {
```

Comment:
case in point: It's syntactically different here too so it's better to put the logic in a function.